scripts/distances/align.py

```diff
@@ -0,0 +1,663 @@
+from collections import Counter
+import math
+import distance
+from munkres import Munkres, print_matrix
+import numpy as np
+from textutil import break2sentences, extract_terminals
+import evaluation
+import re
+import string
+import zss
+from ucca import layer0, layer1
+
+import inspect
+
+def compare(n1, n2):
+	return True if labels(n1) & labels(n2) else False
+
+
+def labels(n):
+	return evaluation.expand_equivalents(set(e.tag for e in n.incoming))
+
+
+def label(n):
+	return n.ftag
+
+
+def is_terminal(n):
```

Comment:
not exactly, there is a type or a tag, but this returns true also tags of the terminals (tags that are not interesting when comparing paragraphs when one word might be split into two for example)